src/otx/cli/cli.py

```diff
@@ -326,41 +329,53 @@ def _set_extension_subcommands_parser(self, parser_subcommands: _ActionSubComman
             )
             parser_subcommands.add_subcommand("find", find_parser, help="This shows the model provided by OTX.")
 
-    def instantiate_classes(self) -> None:
+    def instantiate_classes(self, instantiate_engine: bool = True) -> None:
         """Instantiate the necessary classes based on the subcommand.
 
         This method checks if the subcommand is one of the engine subcommands.
         If it is, it instantiates the necessary classes such as config, datamodule, model, and engine.
+
+        Args:
+            instantiate_engine (bool, optional): Whether to instantiate the engine. Defaults to True.
         """
         if self.subcommand in self.engine_subcommands():
             # For num_classes update, Model and Metric are instantiated separately.
             model_config = self.config[self.subcommand].pop("model")
-            metric_config = self.config[self.subcommand].pop("metric")
+            metric_config = self.config[self.subcommand].get("metric")
```

Comment:
Will it affect to https://github.com/openvinotoolkit/training_extensions/blob/fa70d8c357bf8f7d3b3c7a41532e4053d53430e4/src/otx/cli/cli.py#L374?

Reply:
In the output configuration file, the metric section was currently being dropped; fix this.